vision/module/location.py

```diff
@@ -377,6 +385,23 @@ def showCircles(self) -> None:
         cv2.waitKey(0)
         cv2.destroyAllWindows()
 
+    def saveCircleImage(self, filename, folder) -> None:
+        """
+        Saves image with circles in folder Circle_Images.
+
+        Returns
+        -------
+        None
+        """
+
+        circleImg = np.copy(self.img)
+
+        for x, y, r in self.circles:
+            cv2.circle(circleImg, (x, y), r, (0, 255, 0), 4)
+            cv2.rectangle(circleImg, (x - 5, y - 5), (x + 5, y + 5), (0, 128, 255), -1)
+
+        cv2.imwrite(os.path.join(folder, filename), circleImg)
```

Comment:
Instead of using os.path.join here to add the folder to the filename, you can do that in the benchmark and just pass the whole thing as one parameter to this function.